src/renderer/entities/transaction/lib/extrinsicService.ts

```diff
@@ -454,3 +454,23 @@ export const wrapAsMulti = (
     },
   };
 };
+
+// TODO: finish in later tasks
```

Comment:
which task?

Reply:
Create proxy operation for add/remove proxy #1445 
I will update the comment